src/subdomains/core/liquidity-management/services/liquidity-management-balance.service.ts

```diff
@@ -26,10 +26,9 @@ export class LiquidityManagementBalanceService {
         } catch (e) {
           if (e instanceof BalanceNotCertainException) {
             console.warn(e.message);
-            return null;
           }
 
-          throw e;
+          return null;
```

Comment:
So we just ignore the uncertain asset instead of stopping the process?

Reply:
yes, exactly - it will be caught later as if there is no balance, instead of aborting the whole process

Reply:
Cool, makes sense 👍